lib/signore/tags.rb

```diff
@@ -8,5 +8,9 @@ def match?(sig_tags)
       sig_tags ||= []
       (required & sig_tags) == required and (forbidden & sig_tags).empty?
     end
+
+    def all
+      (required + forbidden.collect { |tag| '~' + tag }).join(' ')
+    end
```

Comment:
This returns a `String` representation of `Tags`, so `#to_s` would be a better name.

Also actually testing this method will show you a bug that hides in the above implementation… :)